client/src/app.test.jsx

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+import { describe, expect, test, vitest } from 'vitest';
```

Comment:
I'm not too familiar with vitest... but if its the same as jest. We can definitely go over how to do some RTL tests. Let me know.

Reply:
I would LOVE to go over this! Thank you for offering Philip.  Yes, vitest is very similar to jest.